src/UI/Buyer/src/app/shared/components/address-display/address-display.component.html

```diff
@@ -8,13 +8,13 @@
       <div>{{address.Street1}}</div>
       <div *ngIf="address.Street2">{{address.Street2}}</div>
       <div>{{address.City}}, {{address.State}} {{address.Zip}}</div>
-      <div *ngIf="address.Phone">{{address.Phone | tel}}</div>
+      <div *ngIf="address.Phone">{{address.Phone}}</div>
     </div>
```

Comment:
We're getting rid of the telephone pipe? :(

Reply:
I know, its my baby. I don't think it will work well without the input formatter though. Now "-------" is technically a valid phone number

Reply:
Could you use both?

Reply:
Archway asked to remove the phone input formatter. It was messing up Canadian phone numbers.

Reply:
Boo tell em to get their own app :downvote: